stockflux-securities-master/src/components/InputForm.js

```diff
@@ -16,6 +16,8 @@ const InputForm = ({ match }) => {
   const [inProgress, setInProgress] = useState(false);
   const [isLoading, setIsLoading] = useState(false);
   const [errorMessage, setErrorMessage] = useState(null);
+  const [sendingMessages, setSendingMessages] = useState([]);
```

Comment:
The naming of `sendingMessages` and `sendingSuccess` doesn't seem right to me. You're also using an awful lot of state for what is effectively a bunch of messages and a type (error/success). I think it would be worth refactoring this to be something like `messages` which would store an array of messages and `type` which would dictate the type that you pass to your `Message` component.

Reply:
Okay all the messages are now contained in the same object with a type: success or error and with a message: [array of messages, ...]